fiftyone/core/collections.py

```diff
@@ -2722,8 +2865,16 @@ def set_label_values(
                 use the default value ``fiftyone.config.show_progress_bars``
                 (None), or a progress callback function to invoke instead
         """
+        if isinstance(values, list):
+            _values = [d["value"] for d in values]
+        else:
+            _values = values.values()
```

Comment:
What are your feelings on using a ternary in situations like this? e.g.
```suggestion
        _values = [d["value"] for d in values] if isinstance(values, list) else values.values()
```